projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/document/write-document-field.component.spec.ts

```diff
@@ -4,11 +4,12 @@ import { FormGroup } from '@angular/forms';
 import { MatDialog, MatDialogConfig, MatDialogRef } from '@angular/material/dialog';
 import { By } from '@angular/platform-browser';
 import { MockComponent } from 'ng2-mock-component';
-import { of, Subscription, throwError } from 'rxjs';
+import { Subscription, of, throwError } from 'rxjs';
```

Comment:
@DavidJayakumar - import ordering... are you sure this is correct? Expected to see `{ of, Subscription, throwError }`

Reply:
It rearragnes atomatically on save. So it seem like correct ordering

Reply:
OK... must be a variation between VSCode and IntelliJ, @DavidJayakumar